sycl/include/sycl/handler.hpp

```diff
@@ -3365,6 +3368,7 @@ class __SYCL_EXPORT handler {
   std::unique_ptr<detail::HostKernelBase> MHostKernel;
 
   detail::code_location MCodeLoc = {};
+  bool MIsTopCodeLoc = true;
```

Comment:
Adding this field breaks ABI of handler (hence the test failure), it should be moved to handler_impl to avoid this.

Reply:
Many thanks sergey. Added commit to fix these ABI issues.